go.mod

```diff
@@ -6,16 +6,13 @@ require (
 	contrib.go.opencensus.io/exporter/jaeger v0.2.0
 	contrib.go.opencensus.io/exporter/ocagent v0.6.0
 	contrib.go.opencensus.io/exporter/zipkin v0.1.1
-	github.com/Masterminds/sprig/v3 v3.0.2 // indirect
 	github.com/chzyer/logex v1.1.10 // indirect
 	github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1 // indirect
-	github.com/huandu/xstrings v1.3.0 // indirect
 	github.com/micro/cli/v2 v2.1.2-0.20200203150404-894195727d9c
 	github.com/micro/go-micro/v2 v2.0.1-0.20200212105717-d76baf59de2e
 	github.com/micro/micro/v2 v2.0.1-0.20200210100719-f38a1d8d5348
-	github.com/mitchellh/reflectwalk v1.0.1 // indirect
 	github.com/openzipkin/zipkin-go v0.2.2
-	github.com/owncloud/ocis-devldap v0.0.0-20200210154914-6c796445c1bf
+	github.com/owncloud/ocis-glauth v0.1.1-0.20200317074939-4e1a5ad5c659
```

Comment:
We should do a new release of glauth first, v0.2.0

Reply:
v0.2.0 is released.

Reply:
now using v0.2.0